core/Cookie.php

```diff
@@ -153,7 +156,8 @@ protected function setCookie($Name, $Value, $Expires, $Path = '', $Domain = '',
             . (empty($Path) ? '' : '; path=' . $Path)
             . (empty($Domain) ? '' : '; domain=' . $Domain)
             . (!$Secure ? '' : '; secure')
-            . (!$HTTPOnly ? '' : '; HttpOnly');
+            . (!$HTTPOnly ? '' : '; HttpOnly')
+            . (!$sameSite ? '' : '; SameSite=' . $sameSite);
```

Comment:
just to be safe we maybe want to rawurlencode the samesite value... and just to be safe maybe also domain and path even though they are values coming from our system?